### ISSUE TITLE
USE_LONG_FOR_INTS not evaluated in UntypedObjectDeserializer

### ISSUE THREAD

**Issue by jarmoni** [FasterXML/jackson-databind#1460]
```
objectMapper.configure(DeserializationFeature.USE_LONG_FOR_INTS, true)
```
has no effect when using 'UntypedObjectDeserializer'. So every Number <= 32 Bit is mapped to Integer .

**Comment by cowtowncoder** [comment]
That'd be wrong. Is this with latest published version(s)?

**Comment by jarmoni** [comment]
I refer to version 2.8.5.

**Comment by cowtowncoder** [comment]
@jarmoni Makes sense, just wanted to verify.

**Comment by cowtowncoder** [comment]
I can not reproduce this problem, but I suspect this might be due to #1453 having fixed this problem as well -- it will be part of 2.8.6. It is included in `2.8` branch. I'll add a test (which passes) to ensure fix works.
